src/PaneDetails/PaneDetails.js

```diff
@@ -152,4 +153,6 @@ export default stripesForm({
   form: 'FormVendor',
   navigationCheck: true,
   enableReinitialize: true,
+  keepDirtyOnReinitialize: true,
+  pure: false,
```

Comment:
Please use `connectedSource={connectedSource}` in `ViewVendor.js` for `<this.connectedPaneDetails`